test/jdk/java/awt/List/KeyEventsTest/KeyEventsTest.java

```diff
@@ -264,9 +265,6 @@ private void doTest() throws Exception {
         boolean isWin = false;
         if (Platform.isWindows()) {
             isWin = true;
```

Comment:
Can be simplified to
```suggestion
        boolean isWin = Platform.isWindows();
```